students/TinaB/lesson03/mailroom.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
+"""
+Goal:
+The Program
+
+Script to pull list of donors and print a thank you letter.
+
+"""
+# list of donors
+donors_list = [
+    ['William Gates, III', 326892.24, 4],
+    ['Mark Zuckerberg', 65982.55, 3],
+    ['Jeff Bezos', 52636.27, 1],
+    ['Paul Allen', 877.33, 2],
+    ['Steven Hawking', 326892.24, 5],
+    ['Justin Timberlake', 999658.25, 3]]
+
+
+def send_thank_you():
+    """ function for sending thank you message"""
+    donors_list_copy = []
+    for donor in donors_list:
+        donors_list_copy.append(donor)
+
+    while True:
+        print()
+        print('Welcome to the Send A Thank You Menu:')
+        thankyou_input = input('\tEnter The Last Nanme of a donor \n'
+                               '\tType "list" for a list of donors, or \n'
+                               '\tType "quit" to quit\n'
+                               '---->  ')
+        # print(thankyou_input)
+        if thankyou_input.isalpha():
+            if thankyou_input.lower().startswith('q'):
+                break
+            elif thankyou_input.lower() == 'list':
+                print_donors_names()
+                thankyou_input = input('Please enter the name of the donor: ')
+                # print(thankyou_input)
+            # search through donors for matching name
+            for idx, donor in enumerate(donors_list_copy):
```

Comment:
I can't see any reason to loop over a copy of the original list here so I don't think you need `donor_list_copy` and can just use the original `donor_list` throughout